### PR TITLE
14850 - allow basic accounts access to transaction route

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #14850:*
https://github.com/bcgov/entity/issues/14850
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/src/pay_api/resources/account.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def post(account_number: str):
 
         # Check if user is authorized to perform this action
         required_roles = [Role.EDITOR.value, Role.VIEW_ALL_TRANSACTIONS.value] if view_all else [Role.EDITOR.value]
-        check_auth(business_identifier=None, account_id=account_number, all_of_roles=required_roles, is_premium=True)
+        check_auth(business_identifier=None, account_id=account_number, all_of_roles=required_roles, is_premium=False)
```
Comment (kialj876) [inline pay-api/src/pay_api/resources/account.py:223]: you can just remove the `is_premium` arg here

**Review by kialj876 — COMMENTED** [review]
Can you also secure this with a new test please?

File: pay-api/src/pay_api/resources/account.py (hunk 1 above)
Comment (Jxio) [reply pay-api/src/pay_api/resources/account.py:223]: 👍

**Review by seeker25 — DISMISSED** [review]
As per Kials request, need a unit test

**Review by seeker25 — APPROVED** [review]

**Pull request merged**